dataactvalidator/filestreaming/schemaLoader.py

```diff
@@ -141,4 +141,5 @@ def loadAllFromPath(cls,path):
             cls.loadFields(key,filepath)
 
 if __name__ == '__main__':
+    configure_logging()
```

Comment:
The general idea is to call `configure_logging` at the entry points to the code base rather than having them set automatically on import. The benefit here is largely around testing (we had been writing all of the logs to disk, effectively silently ignoring them), which often has different logging requirements than normal application runs. More broadly, though, doing configuration at entry points allows the code to be imported/reused without setting global state for everyone else.